### ISSUE TITLE
SC2086(double-quote to prevent..) needs checks for sh PATH

### ISSUE THREAD

**Issue by Kreyren** [koalaman/shellcheck#1946]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2086
- My shellcheck version (`shellcheck --version` or "online"): 0.7.1
- [X] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [X] I tried on shellcheck.net and verified that this is still a problem (web version checked and affected, latest commit not checked by compiling from source)

#### For new checks and feature suggestions
- [X] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh 

export PATH=$GOPATH/bin:$GOROOT/bin:$PATH

```

https://github.com/Uniminin/Ripple-Auto-Installer/blob/2fc2f9a94ecd5d418be81a9b88945d5623123ae3/Main/ripple.sh#L59

#### Here's what shellcheck currently says:

No issues found

#### Here's what I wanted or expected to see:

Double-quote to prevent word splitting

#### Relevants
- https://github.com/Uniminin/Ripple-Auto-Installer/pull/14

**Comment by jwilk** [comment]
Duplicate of #951.

**Comment by Kreyren** [comment]
@jwilk Agree, closing in favor of #951
